extmod/nimble/modbluetooth_nimble.c

```diff
@@ -1281,14 +1281,15 @@ int mp_bluetooth_gattc_discover_descriptors(uint16_t conn_handle, uint16_t start
 }
 
 STATIC int ble_gatt_attr_read_cb(uint16_t conn_handle, const struct ble_gatt_error *error, struct ble_gatt_attr *attr, void *arg) {
-    DEBUG_printf("ble_gatt_attr_read_cb: conn_handle=%d status=%d handle=%d\n", conn_handle, error->status, attr ? attr->handle : -1);
+    uint16_t handle = attr ? attr->handle : (error ? error->att_handle : 0xfff);
```

Comment:
should the final value here be 0xffff (65535)?  same below

Reply:
Yes. Fixed.